clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/ops/servergroup/DeployKubernetesAtomicOperation.groovy

```diff
@@ -115,6 +115,34 @@ class DeployKubernetesAtomicOperation implements AtomicOperation<DeploymentResul
       task.updateStatus BASE_PHASE, "Adding container ${container.name} with image ${container.image}..."
       replicationControllerBuilder = replicationControllerBuilder.addNewContainer().withName(container.name).withImage(container.image)
 
+      if (container.ports) {
+        task.updateStatus BASE_PHASE, "Setting container ports..."
+      }
+
+      container.ports?.forEach {
```

Comment:
Might help readability to put this block inside of the `if (container.ports) {` block (and drop the `?.`). Otherwise, it feels like having two `if (container.ports)` in a row.